accord-core/src/main/java/accord/primitives/Txn.java

```diff
@@ -53,6 +66,16 @@ public boolean isRead()
             return this == Read;
         }
 
+        /**
+         * Note that it is only possible to do this for transactions whose execution time is not dependent on
+         * others, i.e. where we may safely propose executeAt = txnId regardless of when it is witnessed by
```

Comment:
How does a transaction end up being that way? What transaction could safely propose a txnId of 0 at any time?

Also what is "this"? :-)

It's proposing invalidations, don't those need to properly order with what already occurred?

Reply:
It's only possible to "propose deps". I will clean up this comment, to contain similar text to the other proposed places to explain this distinction. In this case we're essentially saying that, because this transaction doesn't actually execute, it doesn't need to agree an `executeAt`, permitting it the option of instead agreeing some `deps` from the single `PreAccept` round (since the deps after `PreAccept` are valid for `executeAt=txnId`, but not for `executeAt>txnId`)